apps/api-documenter/src/yaml/OfficeYamlDocumenter.ts

```diff
@@ -92,17 +92,16 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     if (snippets) {
       delete this._snippets[nameWithoutPackage];
 
-      if (!yamlItem.remarks) {
-        yamlItem.remarks = '';
-      }
-
-      yamlItem.remarks += '\n\n#### Examples\n';
-      for (const snippet of snippets) {
-        if (snippet.search(/await/) === -1) {
-          yamlItem.remarks += '\n```javascript\n' + snippet + '\n```\n';
-        } else {
-          yamlItem.remarks += '\n```typescript\n' + snippet + '\n```\n';
+      if (yamlItem.remarks) {
+        yamlItem.remarks += this._generateSnippetText(snippets);
```

Comment:
>_generateSnippetText [](start = 33, length = 20)

You can clean this code up by generating the snippets before this big conditional block and then deciding where to append them afterward.

Reply:
Ack.